website/docs/guides/relay-resolvers/defining-types.md

```diff
@@ -0,0 +1,49 @@
+---
+id: defining-types
+title: "Defining Types"
+slug: /guides/relay-resolvers/defining-types/
+description: How to define types for your client state schema
+---
+
+You can think of client state resolvers as defining a GraphQL server that runs in the client. Just like with a server-defined GraphQL server you will need to define the _types_ that exist in your schema as well as the _fields_ on those types. Just like a server GraphQL server, fields are defined as functions that compute the GraphQL value from the parent object. In Relay Resolvers we call this parent JavaScript object the "model" of the type.
+
+:::info
+Each client state GraphQL type is backed by a JavaScript object type which these docs will refer to as its "model type". Resolvers "on" this type will be passed an instance of this type as their first argument.
+:::
+
+Resolver types are defined using the `@RelayResolver` tag followed by the name of the type you are defining. By default Relay assumes your client types are “strong”, meaning each instance has a unique ID. This property allows Relay to apply a number of optimizations, such as memoizing resolver computation.
+
+### Defining a “strong” type
+
+Strong types ar defined by a docblock followed by an exported function whose name matches the type's name, and which accepts an ID as its first argument and returns an instance of the type’s model [TODO link]. Resolvers that define edges to this type will simply need to return the ID of the object, rather than deriving the model themselves.
```

Comment:
is it right that the model resolver can only accept one argument (the id argument)? Seems to be the case when I tried it. IIUC, instead of "first" argument we can say "only"?